js/hotgraphicModel.js

```diff
@@ -21,18 +21,17 @@ export default class HotgraphicModel extends ItemsComponentModel {
       const tooltip = child.get('_tooltip');
       if (!tooltip?._isEnabled) return;
       tooltip._id = `hotgraphic-pin-${id}-${index}`;
+      tooltip._classes = (tooltip._classes || '') + ' hotgraphic__pin-tooltip';
```

Comment:
```suggestion
      tooltip._classes = [
        tooltip._classes, 
        'hotgraphic__pin-tooltip'
      ].filter(Boolean).join(' ');
```
This format tends to achieve slightly better white spacing.